ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -639,4 +651,19 @@ public void shouldSetShowDescriptionsForShowQueriesDescriptions() {
     ListQueries listQueries = (ListQueries)statement;
     Assert.assertThat(listQueries.getShowExtended(), is(true));
   }
+  
+    private void assertQuery(Statement statement) {
```

Comment:
nit: indentation

Reply:
Test would also read better if this method was `assertQuerySucceeds("SELECT * FROM BLAH")`.